recipes/nlohmann_json/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "nlohmann_json" %}
+{% set version = "3.1.2" %}
+{% set sha256 = "e8fffa6cbdb3c15ecdff32eebf958b6c686bc188da8ad5c6489462d16f83ae54" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/nlohmann/json/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+  # The json library is header only, so we do not need
+  # to export the compiler run-time libraries.
+  # They are just needed during the build to perform some checks.
+  ignore_run_exports:
+    - libcxx
+    - libgcc-ng
+    - libstdcxx-ng
+    - vc
+    - vs2015_runtime
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/nlohmann  # [unix]
+    - test -f ${PREFIX}/include/nlohmann/json.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/nlohmann_json/nlohmann_jsonConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/nlohmann_json/nlohmann_jsonConfigVersion.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/nlohmann_json/nlohmann_jsonTargets.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\nlohmann\json.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\nlohmann_jsonConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\nlohmann_jsonConfigVersion.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\nlohmann_jsonTargets.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://github.com/nlohmann/json
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.MIT
+  summary: 'JSON for Modern C++'
+  description: 'JSON for Modern C++ is a C++11 JSON parser.'
+  doc_url: http://nlohmann.github.io/json/
+  dev_url: http://github.com/nlohmann/json
+
+extra:
+  recipe-maintainers:
+    - constantinpape
+    - SylvainCorlay
+    - JohanMabille
+    - wolfv
```

Comment:
Added based on [this comment]( https://github.com/conda-forge/staged-recipes/pull/6427#issuecomment-412013497 ).